### PR TITLE
Added 'CLI for Microsoft 365 v7.5' release announcement

### PR DESCRIPTION
**Pull request by Jwaegebaert**
## Category

- [ ] Content fix
- [x] New article

## Contents of the Pull Request

This PR is the release announcement for the v7.5 release of the CLI for Microsoft 365.

### CODE REVIEW

File: content/cli-for-microsoft-365/cli-for-microsoft-365-v7-5/index.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+  - images/banner-cli-m365.png
+tags:
+  - CLI for Microsoft 365
+  - Entra ID
+  - Graph
+  - Viva Engage
+type: popular
+---
+
+Introducing the latest release of CLI for Microsoft 365
```
Comment (waldekmastykarz) [inline content/cli-for-microsoft-365/cli-for-microsoft-365-v7-5/index.md:18]: Is this meant to be a heading? If not, let's merge it with the next paragraph. It seems odd the way it is now.

File: content/cli-for-microsoft-365/cli-for-microsoft-365-v7-5/index.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+
+[CLI for Microsoft 365](https://aka.ms/cli-m365) is a cross-platform command-line tool that allows you to manage your Microsoft 365 tenant and SharePoint Framework projects. This minor release introduces several new commands that will enhance the quality of your Microsoft 365 experience.
+
+> Explore the [release notes](https://aka.ms/cli-m365/notes) to discover an array of exciting features and improvements that will revolutionize your Microsoft 365 journey. 
+ 
+## What's new
+
+### Support for multiple connections
```
Comment (waldekmastykarz) [inline content/cli-for-microsoft-365/cli-for-microsoft-365-v7-5/index.md:26]: I suggest we rephrase it to: "support for using multiple accounts". Connections how we facilitate it internally, but it might be unclear for folks what it means exactly. On the other hand, "multiple accounts" clearly depicts what this feature allows people to do.

File: content/cli-for-microsoft-365/cli-for-microsoft-365-v7-5/index.md [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+
+The release includes a mayor quality-of-life improvement for users who manage multiple tenants. Now you can manage multiple connections to different tenants. This feature allows you to switch between tenants without having to log out and log back in. 
+
+This introduces a whole new group of commands to manage connections. You can now list, remove, set and use different connections. With this new feature we also added the ability to provide a custom name for each connection. If this is not provided, a random GUID will be used as the connection name.
```
Comment (waldekmastykarz) [inline content/cli-for-microsoft-365/cli-for-microsoft-365-v7-5/index.md:30]: I suggest we rephrase to "We introduce..." to make it easier to read

Comment (waldekmastykarz) [inline content/cli-for-microsoft-365/cli-for-microsoft-365-v7-5/index.md:30]: If no name for the connection is specified, we use the localAccountId rather than a random GUID.

File: content/cli-for-microsoft-365/cli-for-microsoft-365-v7-5/index.md [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+
+For more information check the documentation here: [m365 file copy](https://pnp.github.io/cli-microsoft365/cmd/file/file-copy/)
+
+## What's changed
+
+Besides all these new commands, we've also made some changes to the existing commands. A lot of bugs have been fixed and the codebase has been polished. A major change that has been made is the renaming of the `yammer` command group to `viva engage`. This is to better reflect the change Microsoft has made to the name of the service.
+
+Also, a lot of improvements have been made to the documentation. We've added more examples and improved the readability of the documentation and we introduced a new article. This article explains how to use Use CLI for Microsoft 365 VS Code extension. You can find this article [here](https://pnp.github.io/cli-microsoft365/user-guide/using-cli-vs-code-extension/).
+
+If you are eager to go over all of the details and improvements added in this release, do not hesitate to check out the [release notes](https://pnp.github.io/cli-microsoft365/about/release-notes#v740) to find out more.
```
Comment (waldekmastykarz) [inline content/cli-for-microsoft-365/cli-for-microsoft-365-v7-5/index.md:140]: Let's update the link to point to v7.5 release notes

**Review by waldekmastykarz — CHANGES_REQUESTED** [review]
Very nice! Let's do a couple of changes

File: content/cli-for-microsoft-365/cli-for-microsoft-365-v7-5/index.md [hunk 5]
```diff
@@ -0,0 +1,197 @@
...
+
+[CLI for Microsoft 365](https://aka.ms/cli-m365) is a cross-platform command-line tool that allows you to manage your Microsoft 365 tenant and SharePoint Framework projects. This minor release introduces several new commands that will enhance the quality of your Microsoft 365 experience.
+
+> Explore the [release notes](https://aka.ms/cli-m365/notes) to discover an array of exciting features and improvements that will revolutionize your Microsoft 365 journey. 
+ 
+## What's new
+
+### Support for multiple connections
+
+The release includes a mayor quality-of-life improvement for users who manage multiple tenants. Now you can manage multiple connections to different tenants. This feature allows you to switch between tenants without having to log out and log back in.
```
Comment (milanholemans) [inline content/cli-for-microsoft-365/cli-for-microsoft-365-v7-5/index.md:28]:
```suggestion
The release includes a major quality-of-life improvement for users who manage multiple tenants. Now you can manage multiple connections to different tenants. This feature allows you to switch between tenants without having to log out and log back in. 
```

File: content/cli-for-microsoft-365/cli-for-microsoft-365-v7-5/index.md [hunk 6]
```diff
@@ -0,0 +1,197 @@
...
+The release includes a mayor quality-of-life improvement for users who manage multiple tenants. Now you can manage multiple connections to different tenants. This feature allows you to switch between tenants without having to log out and log back in. 
+
+This introduces a whole new group of commands to manage connections. You can now list, remove, set and use different connections. With this new feature we also added the ability to provide a custom name for each connection. If this is not provided, a random GUID will be used as the connection name.
+
+With this major new feature, you'll be able to execute the following commands:
+
+Use the device code flow to connect to a Microsoft 365 tenant and provide a custom name for the connection:
+
+```sh
+m365 login --connectionName "MyConnection"
```
Comment (milanholemans) [inline content/cli-for-microsoft-365/cli-for-microsoft-365-v7-5/index.md:37]: Instead of using abstract `MyConnection`, could we use something more relevant like `Customer A` for example?

File: content/cli-for-microsoft-365/cli-for-microsoft-365-v7-5/index.md [hunk 7]
```diff
@@ -0,0 +1,197 @@
...
+
+And with the second one, you can create a new Microsoft Entra ID group:
+
+```sh
+m365 entra group add --displayName Developers --type microsoft365 --mailNickname devs --ownerUserNames john.doe@contoso.com --memberUserNames "john.doe@contoso.com,adele.vance@contoso.com" --visibility Private
+```
+
+For more information check the following resources:
+- [m365 entra group user add](https://pnp.github.io/cli-microsoft365/cmd/entra/group/group-user-add/)
+- [m365 entra group add](https://pnp.github.io/cli-microsoft365/cmd/entra/group/group-add/)
```
Comment (milanholemans) [inline content/cli-for-microsoft-365/cli-for-microsoft-365-v7-5/index.md:110]: Could we sort them alphabetically?

File: content/cli-for-microsoft-365/cli-for-microsoft-365-v7-5/index.md [hunk 8]
```diff
@@ -0,0 +1,197 @@
...
+
+```sh
+m365 entra user registrationdetails list --userPreferredMethodForSecondaryAuthentication 'sms,push'
+```
+
+For more information check the documentation here: [m365 entra user registrationdetails list](https://pnp.github.io/cli-microsoft365/cmd/entra/user/user-registrationdetails-list/)
+
+### Copy file
+
+We've added a new command to copy a file to a different location. Here you'll be able to copy a file from one location to another within the same site or to a different site. This command is especially useful for users who need to move files around within their SharePoint environment.
```
Comment (milanholemans) [inline content/cli-for-microsoft-365/cli-for-microsoft-365-v7-5/index.md:124]: Let's mention that it uses Graph instead of SP REST APIs.

File: content/cli-for-microsoft-365/cli-for-microsoft-365-v7-5/index.md [hunk 9]
```diff
@@ -0,0 +1,197 @@
...
+m365 file copy --webUrl https://contoso.sharepoint.com/sites/project --sourceUrl "/sites/project/Shared Documents/Document.pdf" --targetUrl "/sites/IT/Shared Documents"
+```
+
+For more information check the documentation here: [m365 file copy](https://pnp.github.io/cli-microsoft365/cmd/file/file-copy/)
+
+## What's changed
+
+Besides all these new commands, we've also made some changes to the existing commands. A lot of bugs have been fixed and the codebase has been polished. A major change that has been made is the renaming of the `yammer` command group to `viva engage`. This is to better reflect the change Microsoft has made to the name of the service.
+
+Also, a lot of improvements have been made to the documentation. We've added more examples and improved the readability of the documentation and we introduced a new article. This article explains how to use Use CLI for Microsoft 365 VS Code extension. You can find this article [here](https://pnp.github.io/cli-microsoft365/user-guide/using-cli-vs-code-extension/).
```
Comment (milanholemans) [inline content/cli-for-microsoft-365/cli-for-microsoft-365-v7-5/index.md:138]:
```suggestion
Also, a lot of improvements have been made to the documentation. We've added more examples, improved the readability of the documentation and we introduced a new article. This article explains how to use Use CLI for Microsoft 365 VS Code extension. You can find this article [here](https://pnp.github.io/cli-microsoft365/user-guide/using-cli-vs-code-extension/).
```

**Review by milanholemans — CHANGES_REQUESTED** [review]
Nice blog @Jwaegebaert! A few small comments from my side.

**Comment by Jwaegebaert** [comment]
Thanks for the review @waldekmastykarz & @milanholemans. All the suggestions have been applied.

**Review by waldekmastykarz — APPROVED** [review]
